pidtree_bcc/plugins/loginuidmap.py

```diff
@@ -0,0 +1,35 @@
+import pwd
+import sys
+
+from pidtree_bcc.plugin import BasePlugin
+
+
+class Loginuidmap(BasePlugin):
+    """ Plugin for mapping PID to loginuid and username """
+
+    NO_LOGINUID = 4294967295  # unsigned -1
+
+    def process(self, event):
+        for proc in event['proctree']:
```

Comment:
Can you just verify that this is actually iterating from the leaf process up to init instead of the other way round?

Also might make sense to add a trapdoor for when the PID for the proc is `1` as we can save a syscall trying to read the virtual file for every process tree - if you don't know who's running your init process I don't think TCP connection tracing is your problem.